examples/plugins/job-pre-post/mail/avocado_job_mail.py

```diff
@@ -12,7 +68,17 @@ class Mail(JobPre, JobPost):
     description = 'Sends mail to notify on job start/end'
 
     def __init__(self):
+        self.enabled = 0
         self.log = logging.getLogger("avocado.app")
+
+    def parse_job_opts(self, job):
+        configured = settings.get_value(section="plugins.job.mail",
+                                        key="configured",
```

Comment:
I think it'd be better to simply ask whether there is any destination email address and assume it's configured in such case. It could come either from config or from cmdline, what do you think?